### PR TITLE
HBW-209 Show spinner only on search

### PR DESCRIPTION
**Pull request by mxsmirnoff**

### CODE REVIEW

**Comment by mxsmirnoff** [comment]
Nov-13-2019 16-11-58

File: hbw/app/javascript/packs/hbw/components/helpers/claiming_tasks.js [hunk 1]
```diff
@@ -51,6 +50,8 @@ export default WrappedComponent => class WithClaimingTasks extends Component {
     const { subscription } = this.state;
     const { env, perPage } = this.props;
 
+    this.setState({ fetching: true });
```
Comment (TimAle) [inline hbw/app/javascript/packs/hbw/components/helpers/claiming_tasks.js]: Let's use context state and do it in `search` function.

File: hbw/app/javascript/packs/hbw/components/claiming/shared/task_list_context.jsx [hunk 2]
```diff
@@ -59,6 +60,7 @@ const withTaskListContext = (WrappedComponent) => {
 
     updateSubscription = () => {
       const { page, searchQuery, tab } = this.state;
+      this.setState({ fetching: true });
```
Comment (TimAle) [inline hbw/app/javascript/packs/hbw/components/claiming/shared/task_list_context.jsx]: Move this to `search`, we shoult rotate spinner only for search action.

**Comment by mxsmirnoff** [comment]
Nov-15-2019 10-10-21

**Pull request merged**